packages/salesforcedx-utils-vscode/src/cli/localCommandExecutor.ts

```diff
@@ -0,0 +1,41 @@
+import { EventEmitter } from 'events';
+import 'rxjs/add/observable/fromEvent';
+import { Command } from '.';
+import { CancellationToken, CommandExecution } from './commandExecutor';
+
+import { Observable } from 'rxjs/Observable';
+
+export class LocalCommandExecution implements CommandExecution {
+  public readonly command: Command;
+  public readonly cancellationToken?: CancellationToken;
+  public readonly processExitSubject: Observable<number | undefined>;
+  public readonly processErrorSubject: Observable<Error | undefined>;
+  public readonly stdoutSubject: Observable<Buffer | string>;
+  public readonly stderrSubject: Observable<Buffer | string>;
+
+  public cmdEmitter: EventEmitter = new EventEmitter();
+  public static readonly EXIT_EVENT = 'exitEvent';
```

Comment:
Why not make these constants in the module/file instead of in the `LocalCommandExecution`? This is not Java. It's possible to have constants outside the file.

Reply:
wasn't sure what the preferred pattern was for Typescript.   And to some extent, I'd prefer that these be used clearly as tied to LocalCommandExecution rather than being an arbitrary constant.  At the point of use in tests, etc. it makes more sense to me to prefix them with LocalCommandExecution.   IMHO

Reply:
@GaryFNC - Sounds reasonable. Let's keep it this way for this one. It does seem better to localize it within the class.